src/java.base/share/classes/sun/security/util/SignatureFileVerifier.java

```diff
@@ -848,15 +848,15 @@ private static int initializeMaxSigFileSize() {
          * in a JAR file.
          */
         Integer tmp = GetIntegerAction.privilegedGetProperty(
-                "jdk.jar.maxSignatureFileSize", 8000000);
+                "jdk.jar.maxSignatureFileSize", 16000000);
         if (tmp < 0 || tmp > MAX_ARRAY_SIZE) {
```

Comment:
Can't `tmp` be `null`? After all, it is an `Integer` object. The rest of your code changes look fine to me.

Reply:
`tmp` will not be null. `GetIntegerAction.privilegedGetProperty()` will return the default value when jdk.jar.maxSignatureFileSize=null

Reply:
You're right. Didn't look close enough.

Reply:
Might make sense to declare `tmp` as an `int` to avoid future confusion and save a few bytecode on repeated implicit unboxing every time `tmp` is used.

Reply:
Done as suggested.